### PR TITLE
Jhs tw 1819

### PR DESCRIPTION
**Pull request by JHStoops**
image

Enabled ellipsis on overflow.

### CODE REVIEW

File: src/fs-person.html [hunk 1]
```diff
@@ -315,7 +341,7 @@
...
...mina","fs.shared.fsPerson.MALE":"Maschio","fs.shared.fsPerson.UNKNOWN":"Sesso sconosciuto","fs.shared.fsPerson.UNKNOWN_NAME":"[Nome sconosciuto]"},"ja":{"_LRM_PKTAG688":"fs-person_1_104 - FULL-FILE","fs.shared.fsPerson.FEMALE":"女性","fs.shared.fsPerson.MALE":"男性","fs.shared.fsPerson.UNKNOWN":"性別不明","fs.shared.fsPerson.UNKNOWN_NAME":"〔不明な名前〕"},"ko":{"_LRM_PKTAG400":"fs-person_1_105 - FULL-FILE","fs.shared.fsPerson.FEMALE":"여성","fs.shared.fsPerson.MALE":"남성","fs.shared.fsPerson.UNKNOWN":"성별 미상","fs.shared.fsPerson.UNKNOWN_NAME":"[이름 미상]"},"pt":{"_LRM_PKTAG480":"fs-person_1_106 - FULL-FILE","fs.shared.fsPerson.FEMALE":"Feminino","fs.shared.fsPerson.MALE":"Masculino","fs.shared.fsPerson.UNKNOWN":"Sexo desconhecido","fs.shared.fsPerson.UNKNOWN_NAME":"[Nome desconhecido]"},"ru":{"_LRM_PKTAG394":"fs-person_1_107 - FULL-FILE","fs.shared.fsPerson.FEMALE":"Женский","fs.shared.fsPerson.MALE":"Мужской","fs.shared.fsPerson.UNKNOWN":"Пол неизвестен","fs.shared.fsPerson.UNKNOWN_NAME":"[Неизвестное имя]"},"zh":{"_LRM_PKTAG240":"fs-person_1_108 - FULL-FILE","fs.shared.fsPerson.FEMALE":"女","fs.shared.fsPerson.MALE":"男","fs.shared.fsPerson.UNKNOWN":"性別不詳","fs.shared.fsPerson.UNKNOWN_NAME":"〔姓名不詳〕"}});
```
Comment (justincy) [inline src/fs-person.html]: I don't believe this is supposed to be here. It looks like code was copied from the file `fs-person.html` which is built based on this file.

**Comment by justincy** [comment]
So when looking at the diff, your 1-line change is shown in `fs-person.html` while porting Jason's changes into src is represented in `src/fs-person.html`.

**Comment by JHStoops** [comment]
right, I ported over Jason's code and changed the one line in his code, then built it so it reflected in the fs-person.html file as that single line in src/fs-person.html. 
Also, out of habit I added the two files to the .gitignore file; I just realized you might not want them there. Do you want me to revert that back?

**Comment by justincy** [comment]
You can keep the `package-lock.json` entry but you ought to remove `analysis.json` since it's in the repo already.

**Comment by JHStoops** [comment]
I just took out analysis.json from .gitignore and pushed.

**Comment by justincy** [comment]
Good work!

**Pull request merged**